docs/local-development.md

```diff
@@ -0,0 +1,23 @@
+# Local Development
+
+## create-dao development
+
+1.  Run `yarn` to install packages
+2.  In one terminal
+3.  Run `yarn chain` for local ETH test accounts
+4.  In another terminal
+5.  Run `yarn deploy` to deploy the smart contract
+6.  Run `yarn dev` for next development
```

Comment:
I think it would be great to consolidate into a single script, I didn't try this for this PR because it was outside the scope of it.  Please do open a issue for it